.gitignore

```diff
@@ -12,6 +12,9 @@
 .env.test.local
 .env.production.local
 
+coverage
+package-lock.json
```

Comment:
This project is using a yarn.lock

Reply:
For whatever reason, yarn wasn't installing all of the dependencies I was needing to run the iOS simulator.  This was before the patch you put forward, so I'm not sure if it is still happening.  Happy to ignore this as yarn is so much faster anyhow.